### PR TITLE
[KOGITO-1036] - Extract kogito-images from kogito-cloud repository

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by spolti** [comment]
it does make sense, let me do this.

**Comment by spolti** [comment]
@ricardozanini the disadvantage of doing this is that a lot of unrelated commits will come as well.
most of them related with the images.

**Comment by spolti** [comment]
maybe drop unrelated commits from this one.

**Comment by spolti** [comment]
@sutaakar @ricardozanini @evacchi 
The problem is that by removing commits the original commit id can be changed during a interactive rebase, thoughts?

**Comment by spolti** [comment]
@cristianonicolai do you know where is the jenkins pipeline so we can remove kogito-cloud from there?

**Comment by spolti** [comment]
@ricardozanini @sutaakar @evacchi ok, removed all unrelated commits.
This PR is ready to go.

The only missing thing now, is to remove the kogito-cloud step from pipeline and make sure that kogito-cloud-services from this repo will be built and published on maven. @cristianonicolai can you handle this?

File: kogito-cloud-services/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <artifactId>kogito-bom</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <modules>
+    <module>kogito-cloud-workitems</module>
```
[the remark this answers is not in the document]
Comment (spolti) [reply kogito-cloud-services/pom.xml]: this formatting stuff is not right, let me fix that.

[the remark this answers is not in the document]
Comment (spolti) [reply kogito-cloud-services/pom.xml]: formatting changes fixed.

[the remark this answers is not in the document]
Comment (spolti) [reply kogito-cloud-services/pom.xml]: project orders updated.

File: kogito-cloud-services/.gitignore [hunk 2]
```diff
@@ -0,0 +1,32 @@
+bin/
```
[the remark this answers is not in the document]
Comment (spolti) [reply kogito-cloud-services/.gitignore]: I just bring the commits from kie-cloud repo, will remove it.

**Comment by spolti** [comment]
@cristianonicolai can you please review again?

**Comment by spolti** [comment]
> for the Jenkins build failure, it seems that Jenkins does not like to take the Jenkinsfile from the PR:
> 
> ```
> Loading trusted files from base branch master at 050bc0181fa645f5dc758206d9e9e1fcc192c1cf rather than fec4fa66f8f4945c81fe7eba8fdeb653df45189d
> Obtained Jenkinsfile from 050bc0181fa645f5dc758206d9e9e1fcc192c1cf
> ‘Jenkinsfile’ has been modified in an untrusted revision
> ```

so, the Jenkins file used to test the PR us from master.

**Comment by radtriste** [comment]
yes. I think this is a security issue

File: pom.xml [hunk 3]
```diff
@@ -2,1331 +2,1332 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
```
[the remark this answers is not in the document]
Comment (spolti) [reply pom.xml]: correct, removed the formatting changes.

**Comment by MarianMacik** [comment]
@spolti I have triggered the build using Replay functionality on Jenkins where you can alter the Jenkinsfile (I deleted the kogito-cloud-operator step). This option should be available to everybody with credentials to Jenkins.

**Comment by spolti** [comment]
> @spolti I have triggered the build using Replay functionality on Jenkins where you can alter the Jenkinsfile (I deleted the kogito-cloud-operator step). This option should be available to everybody with credentials to Jenkins.

did you mean 'kogito-cloud'

**Comment by cristianonicolai** [comment]
@spolti thanks!

**Comment by MarianMacik** [comment]
> > @spolti I have triggered the build using Replay functionality on Jenkins where you can alter the Jenkinsfile (I deleted the kogito-cloud-operator step). This option should be available to everybody with credentials to Jenkins.
> 
> did you mean 'kogito-cloud'
@spolti Yes, sorry for typo :)

**Pull request merged**